### ISSUE TITLE
9.4.1.2/5.2 The "Activity settings" button has been correctly implemented using the "aria-expanded" attribute, but the accessible name of the button is always the same: aria-label="Open settings menu", which could be confusing for users. The accessible name should be "Activity settings."

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42235]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v5-n2

9.4.1.2/5.2 The "Activity settings" button has been correctly implemented using the "aria-expanded" attribute, but the accessible name of the button is always the same: aria-label="Open settings menu", which could be confusing for users. The accessible name should be "Activity settings."

**Comment by szaimen** [comment]
Cc @JuliaKirschenheuter but right

**Comment by JuliaKirschenheuter** [comment]
this is activity issue, not resolved until now

**Comment by ShGKme** [comment]
Duplication of https://github.com/nextcloud/server/issues/41901
